### PR TITLE
[enterprise-4.10] Created new sections for eus2eus via web console and cli

### PR DESCRIPTION
**Pull request by obrown1205**
Cherry Picked from <4fda78b> xref: https://github.com/openshift/openshift-docs/pull/56680

Version(s):
4.10

Issue:
https://issues.redhat.com/browse/OSDOCS-3751

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by obrown1205** [comment]
I removed a prereq that suggests:
"Ensure that you are familiar with version-specific prerequisites, such as link:https://docs.openshift.com/container-platform/4.12/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare[administrator acknowledgement] that is required prior to updating from {product-title} 4.10 to 4.12." 

Page does not exist in 4.11 or 4.9. Suggestions? Can I remove?

**Comment by shellyyang1989** [comment]
@evakhoni @yapei for review. Thank you!

**Comment by yapei** [comment]
web console changes look good to me

File: modules/updating-eus-to-eus-upgrade-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+----
+
+. Update to version 4.10 by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-latest
+----
+
+. Retrieve the cluster version to ensure that the 4.12 updates are complete by running the following command:
```
[the remark this answers is not in the document]
Comment (obrown1205) [reply modules/updating-eus-to-eus-upgrade-cli.adoc:104]: on it! one sec

**Review by adellape — APPROVED** [review]

**Pull request merged**